src/main/java/com/jfb/lecture5/Main.java

```diff
@@ -1,28 +1,51 @@
 package com.jfb.lecture5;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.jfb.lecture5.model.BusTicket;
+import com.jfb.lecture5.model.Validator;
 
-import java.util.Scanner;
+import java.io.*;
+import java.util.ArrayList;
 
-public class Main {
-  public static void main(String[] args) throws JsonProcessingException {
-    int x = 0;
-
-    do {
-      String input = getInput();
-      BusTicket busTicket = new ObjectMapper().readValue(input, BusTicket.class);
-
-      // TODO: ticket validation
 
-      System.out.println(busTicket.toString());
-      x++;
-
-    } while (x < 5);
+public class Main {
+  public static void main(String[] args) throws IOException, IllegalAccessException {
```

Comment:
try to handle these exceptions (IOException, IllegalAccessException), otherwise the app will be stopped if at least 1 exception occurs. It is better to handle IOException in method where you read the file.

For example, you can catch them and print info about it.